contrib/bundle/vars

```diff
@@ -6,7 +6,7 @@ GIT_ROOT=$(git rev-parse --show-toplevel)
 ARCHIVE_PATH="$GIT_ROOT/build/bundle"
 mkdir -p "$ARCHIVE_PATH"
 
-BUNDLE=crio-$(git describe --tags --exact-match 2>/dev/null || git rev-parse --short HEAD)
+BUNDLE=crio-$(git describe --tags --exact-match 2>/dev/null || git rev-parse HEAD)
```

Comment:
@harche I have to change the bundle name once again in https://github.com/cri-o/cri-o/pull/4552.

My idea is to name the directory within the tarball always `cri-o` (no hash) and amend the architecture to the bundle, like: `cri-o.arm64.$SHA.tar.gz`

Reply:
sounds good, in that case how would the complete download url would look like? I want to make sure I got that correctly to update https://github.com/kubernetes/test-infra/pull/20898

So if you are doing that in https://github.com/cri-o/cri-o/pull/4552, do we still need to have this PR open?

Reply:
I don't think that wee need this PR since I have to touch both lines in any case. :)

After this got merged, the complete download URL should look like:
```
https://storage.googleapis.com/k8s-conform-cri-o/artifacts/cri-o.amd64.$(git ls-remote https://github.com/cri-o/cri-o master | awk '{print $1}').tar.gz
```

Not sure if we can avoid the `awk` invocation.

Reply:
Ah maybe just by using `cut -f1` instead of `awk`.

Reply:
Alright, then I will close this PR and update https://github.com/kubernetes/test-infra/pull/20898 with the new url and put that one on the hold until https://github.com/cri-o/cri-o/pull/4552 is merged.